### PR TITLE
Finish styling and functions of notification page

### PR DESCRIPTION
**Pull request by racecn**
Added:
- tailwind styling of notifs page
- applied the workings of rishitha's alert system in the sidebar

Fixed:
- changed the timestamp system to correctly convert the timestamp if they are within one minute of being created

### CODE REVIEW

File: src/components/NavigationBar.js [hunk 1]
```diff
@@ -37,7 +72,7 @@ export default function NavigationBar() {
            />
         </Link>
         <Link to="/notifications" data-tip="Notifications" data-for="notifications-tooltip">
-          <SideBarItem icon={<FontAwesomeIcon icon={faBell}/>} text="Notifications" />
+          <SideBarItem icon={<FontAwesomeIcon icon={faBell}/>} text="Notifications"  alert={newNotifications} />
```
Comment (rishithab48) [inline src/components/NavigationBar.js:75]: Is the alert working well?

Comment (racecn) [reply src/components/NavigationBar.js:75]: It does! it looks nice too, if you check out the branch I added a test button to the notification page so you can see how it works

Comment (rishithab48) [reply src/components/NavigationBar.js:75]: I just checked it. It looks great!

**Review by rishithab48 — APPROVED** [review]
Looks great!

**Pull request merged**